### ISSUE TITLE
Drop old ffmpegs

### ISSUE THREAD

**Issue by grahamc** [NixOS/nixpkgs#22372]
ffmpeg 1.x and 0.x aren't maintained anymore.

Things depending on 1.x:

 - accousticFingerprinter
 - baresip
 - freerdpStable

Things depending on 0.10:

 - spotify
 - ultrastardx
 - gst_ffmpeg*

**Comment by peterhoeg** [comment]
freerdpStable can be dropped - it's so bugridden that it's practically unusable.

gst_ffmpeg is part of legacy gstreamer which is still in use by quite a few packages.

**Comment by grahamc** [comment]
Yeah, this ticket is more aspirational than anything else. Maybe in a year someone will say "(triage) no activity in a year, closing" and then I can say "but... but..." and then maybe I'll be able to drop at least 1.x. :P (hi, future contributor.)

**Comment by FRidh** [comment]
`ffmpeg` isn't mentioned in the `deb` file of Spotify as a requirement, so I guess we don't need it either. I just 'rebuild' Spotify without and it seems to be working just fine.

**Comment by retrry** [comment]
I looked at Spotify package in AUR[1] and it seems that ffmpeg is used for playing local files.

[1] https://aur.archlinux.org/packages/spotify/

**Comment by cpages** [comment]
@FRidh it is used for local playback. I did a quick test to see if it runs with the default ffmpeg version, but it doesn't. So I guess there's not much we can do for now :/

**Comment by FRidh** [comment]
ah, local playback wasn't part of my test :)
